data/clients.json

```diff
@@ -147,7 +147,7 @@
     },
     {
         "doap": null,
-        "last_renewed": null,
+        "last_renewed": "2013-04-25T00:00:00",
```

Comment:
The idea is not to simply list all clients. Ideally, the developer update here themselves. Hence, I would not like to update this here.

Reply:
Sorry, I just see you are setting old time stamps - sorry for that!